ui/packages/catalog-ui-search/src/main/webapp/component/query-basic/query-basic.view.js

```diff
@@ -543,7 +543,10 @@ module.exports = Marionette.LayoutView.extend({
     }
 
     const types = this.basicType.currentView.model.getValue()[0]
-    const specificTypes = this.basicTypeSpecific.currentView.model.getValue()[0]
+    const specificTypes =
+      this.basicTypeSpecific.currentView.model !== undefined
+        ? this.basicTypeSpecific.currentView.model.getValue()[0]
```

Comment:
❓ Can `getValue()` return an empty array?

Reply:
i think the only way that could happen is if someone entered an empty string in the catalog ui search config